src/views/catalog/CreateFromInstanceTypes/state/useInstanceTypeVMStore.ts

```diff
@@ -0,0 +1,39 @@
+import { useEffect } from 'react';
+
+import { isEqualObject } from '@kubevirt-utils/components/NodeSelectorModal/utils/helpers';
+import { DEFAULT_NAMESPACE } from '@kubevirt-utils/constants/constants';
+import { ALL_NAMESPACES_SESSION_KEY } from '@kubevirt-utils/hooks/constants';
+import { useActiveNamespace } from '@openshift-console/dynamic-plugin-sdk-internal';
+
+import useBootableVolumes from './hooks/useBootableVolumes';
+import useInstanceTypesAndPreferences from './hooks/useInstanceTypesAndPreferences';
+import { useInstanceTypeVMInitialStore } from './hooks/useInstanceTypeVMInitialStore';
+
+export const useInstanceTypeVMStore = () => {
+  const store = useInstanceTypeVMInitialStore();
+  const bootableVolumesData = useBootableVolumes();
+  const instanceTypesAndPreferencesData = useInstanceTypesAndPreferences();
+
+  const [activeNamespace] = useActiveNamespace();
+
+  useEffect(() => {
+    if (!isEqualObject(store.bootableVolumesData, bootableVolumesData)) {
+      store.setBootableVolumesData(bootableVolumesData);
+    }
+  }, [bootableVolumesData, store]);
+  useEffect(() => {
```

Comment:
nitpick: Would you mind putting an empty line between these `useEffect` blocks to make it easier to read?